File: VB/E911.Module.Win/Controllers/WinShowCustomFormWindowController.vb
Imports Microsoft.VisualBasic
Imports System
Imports E911.Module.Controllers
Imports System.Windows.Forms
Imports E911.Module.Editors

Namespace E911.Module.Win.Controllers
	Public Class WinShowCustomFormWindowController
		Inherits ShowCustomFormWindowController
		Protected Overrides Sub ShowCustomForm()
			Dim form As Form = TryCast(DevExpress.Persistent.Base.ReflectionHelper.CreateObject("E911.Module.Win.Controls.WinCustomForm"), Form)
			'Initializing a form when it is invoked from a controller.
			Dim TempXpoSessionAwareControlInitializer As XpoSessionAwareControlInitializer = New XpoSessionAwareControlInitializer(TryCast(form, IXpoSessionAwareControl), Application)
			form.Show()
		End Sub
	End Class
End Namespace

File: CS/E911.Module.Win/Controllers/WinShowCustomFormWindowController.cs
using System;
using E911.Module.Controllers;
using System.Windows.Forms;
using E911.Module.Editors;

namespace E911.Module.Win.Controllers {
    public class WinShowCustomFormWindowController : ShowCustomFormWindowController {
        protected override void ShowCustomForm() {
            Form form = DevExpress.Persistent.Base.ReflectionHelper.CreateObject(
                    "E911.Module.Win.Controls.WinCustomForm") as Form;
            //Initializing a form when it is invoked from a controller.
            new XpoSessionAwareControlInitializer(form as IXpoSessionAwareControl, Application);
            form.Show();
        }
    }
}